scripts/falco-driver-loader

```diff
@@ -711,6 +707,13 @@ if [ -z "$source_only" ]; then
 		exit 1
 	fi
 
+	get_target_id
+	res=$?
+	if [ $res != 0 ]; then
+		>&2 echo "Detected an unsupported target system, please get in touch with the Falco community. Trying to compile anyway."
```

Comment:
`Trying to compile anyway` is true only if `ENABLE_COMPILE` is yes.

Perhaps it is worth printing a different message if both compile&download are disabled, and give up immediately.

Reply:
That could be a great idea; but i didn't want to embed more logic inside the script :) 
If this works for you, i'd leave it as is; otherwise i can make the small patch.

All in all, i am hoping for `falco-driver-loader` to be kicked out by `falcoctl` sooner or later!